ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js

```diff
@@ -16,6 +15,7 @@ import {
   GAS_RECOMMENDATIONS,
 } from '../../../../shared/constants/gas';
 import { getGasFormErrorText } from '../../../helpers/constants/gas';
+import { networkAndAccountSupports1559 } from '../../../selectors';
```

Comment:
nit: can we call the function with a verb as the first thing? E.g. `doesNetworkAndAccountSupport1559`.

Another question: in Swaps we use the `isEIP1559Network` call in a few places. Can we keep using it there or should we call `doesNetworkAndAccountSupport1559` instead?

Reply:
1.  Updated.
2.  Yes, you'll need to switch to the new function, because hardware wallets wont support 1559 our of the gate.